frontend/src/app/app-routing.module.ts

```diff
@@ -1,14 +1,17 @@
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
-import { AdminTableComponent } from './admin-table/admin-table.component'
+import {NgModule} from '@angular/core';
```

Comment:
```suggestion
import { NgModule } from '@angular/core';
```
Follow Angular templates for imports. Applies to all imports.